lib/fhir_types/src/date_time_picker.dart

```diff
@@ -73,6 +74,7 @@ class _FhirDateTimePickerState extends State<FhirDateTimePicker> {
     if (widget.pickerType != Time) {
       final date = await showDatePicker(
         initialDate: _dateTimeValue?.value ?? DateTime.now(),
+        initialEntryMode: QuestionnaireTheme.of(context).datePickerEntryMode,
```

Comment:
Just to keep the consistency, I think it would be better to pass QuestionnaireTheme.of(context).datePickerEntryMode from parent widget, e.g. DateTimeAnswerFiller. In this widget, we accept datePickerEntryMode as a parameter.

Please correct me if I am wrong, by doing this, it also makes things more predictable?

Reply:
Thanks a lot. I also thought reading the QuestionnaireTheme parameters here was a bit off while implementing this, but didn't get to explore this much further.

Your suggestion makes a lot of sense, and I've already pushed the changes. Please let me know if that's ok with you.